### PR TITLE
V3-183016745 empty case table

### PR DESCRIPTION
**Pull request by eireland**
Shows an empty case table on load

### CODE REVIEW

File: v3/src/components/app.tsx [hunk 1]
```diff
@@ -47,10 +47,22 @@
     }
   }
 
+  function createNewTable() {
```
Comment (bfinzer) [inline v3/src/components/app.tsx:50]: Suggest createNewStarterDataset since the table appears as a side effect

File: v3/src/components/app.scss [hunk 2]
```diff
@@ -21,6 +21,12 @@
       align-items: center;
       justify-content: center;
     }
+
+    .add-new-table-button {
```
Comment (bfinzer) [inline v3/src/components/app.scss:25]: This style isn't used, right?

**Review by bfinzer — CHANGES_REQUESTED** [review]

**Comment by bfinzer** [comment]
To help us distinguish V3 pull requests from V2 pull requests, we start the name with "V3—" and we use the magenta V3 label

File: v3/src/components/case-table/use-rows.ts [hunk 3]
```diff
@@ -55,7 +55,7 @@ export const useRows = (data?: IDataSet) => {
             const strValue = data.getValue(caseId, attr.id)
             const numValue = data.getNumeric(caseId, attr.id)
             const formatStr = attr.format || kDefaultFormatStr
-            const formatted = (numValue != null) && isFinite(numValue) ? format(formatStr)(numValue) : strValue;
+            const formatted = (numValue != null) && isFinite(numValue) ? format(formatStr)(numValue) : strValue
```
Comment (kswenson) [inline v3/src/components/case-table/use-rows.ts:58]: FYI, the way the semicolon got in there was that semicolons can be required depending on how the next line begins. That used to be true here, but then the code for the subsequent line was changed without removing the semicolon.

**Review by kswenson — APPROVED** [review]
👍 Nothing to add beyond Bill's comments

**Review by bfinzer — APPROVED** [review]
👍🏻 LGTM

**Pull request merged**